swarm/swarm.go

```diff
@@ -66,20 +67,22 @@ var (
 
 // the swarm stack
 type Swarm struct {
-	config      *api.Config        // swarm configuration
-	api         *api.API           // high level api layer (fs/manifest)
-	dns         api.Resolver       // DNS registrar
-	fileStore   *storage.FileStore // distributed preimage archive, the local API to the storage with document level storage/retrieval support
-	streamer    *stream.Registry
-	bzz         *network.Bzz       // the logistic manager
-	backend     chequebook.Backend // simple blockchain Backend
-	privateKey  *ecdsa.PrivateKey
-	corsString  string
-	swapEnabled bool
-	netStore    *storage.NetStore
-	sfs         *fuse.SwarmFS // need this to cleanup all the active mounts on node exit
-	ps          *pss.Pss
-	swap        *swap.Swap
+	config       *api.Config        // swarm configuration
+	api          *api.API           // high level api layer (fs/manifest)
+	dns          api.Resolver       // DNS registrar
+	fileStore    *storage.FileStore // distributed preimage archive, the local API to the storage with document level storage/retrieval support
+	streamer     *stream.Registry
+	bzz          *network.Bzz       // the logistic manager
+	backend      chequebook.Backend // simple blockchain Backend
+	privateKey   *ecdsa.PrivateKey
+	corsString   string
+	swapEnabled  bool
+	netStore     *storage.NetStore
+	sfs          *fuse.SwarmFS // need this to cleanup all the active mounts on node exit
+	ps           *pss.Pss
+	swap         *swap.Swap
+	stateStore   *state.DBStore
+	metricsStore *leveldb.DB
```

Comment:
I think we should rename this to `accountingMetricsStore` (even though I hate long names), because `metricsStore` doesn't really reflect what this is.